README.de.md

```diff
@@ -0,0 +1,97 @@
+- **Kollaborativ**: Alle Einträge aus einer Klasse sind für alle verfügbar.
+- **Strukturiert**: Dlool hat ein strukturiertes Eingabesystem. Es ist einfach zu bedienen, aber auch sehr leistungsfähig.
+- *(***Exportierbar**: Du können Ihre Daten nach ical oder todo.txt exportieren,
+  mit [diesem] (https://github.com/Dlurak/dlool-scriptable/blob/main/reminders/README.md) Skript können Du Dlool auch mit Apple Reminders synchroniDuren.)* **(noch nicht implementiert)**
+- **Plattformübergreifend**: Zugriff auf Dlool von allen Geräten mit einem Browser.
+- **DunkelmoDus**: Dlool passt sich automatisch an Ihre Systemeinstellungen an. Dlool respektiert auch reDuzierte Bewegungseinstellungen.
+- **REST-API**: Es gibt eine offene REST-API, um eigene Skripte zu schreiben.
+- **Kein Konto erforderlich**: Auch ohne Konto können Du Dlool nutzen.
+- **Langfristige Nutzbarkeit**: Halten Du Ihre Daten über Jahre hinweg organiDurt.
+- **Moderationssystem**: Erhalten Du eine positive, einladende und sichere Umgebung aufrecht.
+- **Notizen**: Schreiben Du kurze Notizen für sich oder Ihre Klasse.
+- **Kalender**: Behalten Du den Überblick über Tests und Fristen mit dem wunderschönen Kalender von Dlool.
+- **Stundenplan**: Mit der Stundenplanunterstützung weißt Du ganz einfach, welche Fächer wann sind und kannst die Hausaufgaben richtig aufschreiben.
+- **Feiertage**: Schauen Du nach, wann in Ihrer Gegend Feiertage sind, die Daten werden von [der OpenHolidayAPI](https://www.openholidaysapi.org/en/) bereitgestellt
+  
+- Viele kleine Dinge, die Ihnen das Leben leichter machen.
+
+---
+
+## Technologien
+
+### Frontend
+
+Dlool ist eine progressive WebanwenDung und wird von den folgenden Technologien unterstützt:
+
+- [SvelteKit](https://kit.svelte.dev/)
+- [TailwindCSS](https://tailwindcss.com/)
+- [Vercel](https://vercel.com/)
+
+### [Backend](https://github.com/Dlurak/dlool_backend_v2)
+
+Das [Backend](https://github.com/Dlurak/dlool_backend_v2) von Dlool wird von den folgenden Technologien unterstützt:
+
+- [Bun](https://bun.sh)
+- [Elysia](https://elysiajs.com/)
+- [EdgeDB](https://www.edgedb.com/)
+- [Render](https://render.com/)
+
+---
+
+## Erste Schritte
+
+Der einfachste Weg, Dlool auszuprobieren, ist ein Besuch auf [der offiziellen Website](https://dlool.me/).
+
+Wenn Du Dlool lokal ausführen möchten, z.B. für die Entwicklung, können Du den Anweisungen unten folgen. ([[#Lokal ausführen]]).
+
+Die Hausaufgaben sind für jeden öffentlich sichtbar, dies ist beabsichtigt, um Mobbing vorzubeugen. Wenn Du aber Hausaufgaben erstellen, aktualiDuren oder löschen willst, brauchst Du einen Account für eine bestimmte Klasse. Du kannst einen Account [hier](https://dlool.me/register) erstellen. Es kann vorkommen, dass jemand aus deiner Klasse bereits Dlool benutzt, dann muss er deine Anfrage, sich für die Klasse anzumelden, akzeptieren. Dies kannst Du tun, indem Du [diese Seite](https://www.dlool.me/moderation/list) aufrufst und auf die Schaltfläche „Akzeptieren“ klickst.
```

Comment:
"aktualiDuren"
In this case your search and replace again didn't quite worked out. It missed a lot like "deine" broke "aktualisieren" and created sentences that are grammatically incorrect